app/views/provider_interface/application_choices/index.html.erb

```diff
@@ -1,6 +1,14 @@
 <% content_for :title, 'Applications' %>
 <% content_for :page_width, 'wide' %>
 
+<% if FeatureFlag.active?('covid_19') %>
+  <div class="app-banner" aria-labelledby="success-message" data-module="govuk-error-summary" role="alert">
```

Comment:
Can we make this 2/3rds width by wrapping in:

```
<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
    
  </div>
</div>
```

Reply:
(The line length makes it a bit hard to read)

Reply:
thanks @fofr, I've wrapped the banner. Do you think the grid belongs in the wrapper to so that it's aligned?